sdk/core/azure-core/test/ut/file_upload.cpp

```diff
@@ -11,7 +11,7 @@
 #include <Windows.h>
 #endif // Windows
 
-#include "azure/core/http/http.hpp"
+#include "azure/core/response.hpp"
```

Comment:
How come this didn't cause CI to fail earlier?

Reply:
This was an old file with an old include.  Not all of response is used and response includes http.hpp.